cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js

```diff
@@ -1,120 +0,0 @@
-/*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
-
-SPDX-License-Identifier: Apache-2.0
-*/
-
-import axios from 'axios';
-
-const routerCreateInvitationPath = `/connections/create-invitation`
-
-/**
- * AgentMediator provides aries mediator features
- * @param aries agent instance
- * @class
- */
-export class AgentMediator {
```

Comment:
Why mediator is deleted?

Reply:
We don't need it. The mediator is accessible through `vuex` actions.
https://github.com/trustbloc/edge-agent/blob/master/cmd/user-agent/src/store/modules/mediator.js#L39